less/viewdetails.less

```diff
@@ -558,7 +558,7 @@
 .document-summary{
   color: #626262;
   border: 1px solid #D8D8D8;
-  border-left: 0.5em solid #5FE45F;
+  border-left: 0.5em solid #A9D361;
```

Comment:
You can use the color variable here:
https://github.com/c2corg/v6_ui/blob/master/less/variables.less#L12